### PR TITLE
add abs(dxy) plot in tracking DQM

### PR DESCRIPTION
**Pull request by mtosi**
add |dxy| plot to tracking DQM

### CODE REVIEW

**Comment by dmitrijus** [comment]
+1

File: DQM/TrackingMonitor/src/TrackAnalyzer.cc [hunk 1]
```diff
@@ -977,6 +987,7 @@ void TrackAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSe
     const reco::BeamSpot& bs = *recoBeamSpotHandle;
 
     DistanceOfClosestApproachToBS      -> Fill(track.dxy(bs.position()));
+    AbsDistanceOfClosestApproachToBS   -> Fill(abs(track.dxy(bs.position())));
```
Comment (davidlange6) [inline DQM/TrackingMonitor/src/TrackAnalyzer.cc]: @mtosi - please use std::abs

Comment (VinInn) [inline DQM/TrackingMonitor/src/TrackAnalyzer.cc]: indeed, look a bit binned at the moment
screen shot 2016-06-10 at 12 54 14 pm

Comment (VinInn) [inline DQM/TrackingMonitor/src/TrackAnalyzer.cc]: in any case 120 bins are most probably more than enough...

**Comment by VinInn** [comment]
@cmsbuild, please test

**Comment by VinInn** [comment]
backport as well please. Hope we can get in next 8_0_X patch

**Comment by mtosi** [comment]
same updates have been already done in PR https://github.com/cms-sw/cmssw/pull/14826 (80x)

**Pull request merged**